setup.py

```diff
@@ -1,11 +1,3 @@
-from pathlib import Path
-
 from setuptools import setup
 
-scripts = [
-    str(filename)
-    for filename in Path("./scripts").glob("**/*")
-    if filename.is_file() and filename.name != "__pycache__"
-]
-
-setup(scripts=scripts)
+setup()
```

Comment:
@zacharyburnett is the `setup.py` even needed anymore? This seems to have been added back by #76.

Reply:
It does not seem to be needed (as per the `setuptools` and `pip` docs) now that we are using PEP621 and PEP518. There used to be guidance to include a "shim" `setup.py` but that seems to have been removed.

Reply:
Whatever the oldest version of python we're supporting (3.9), the current conda distribution of the latest 3.9.x patch (and the one from python.org) should include a version of setuptools and pip that comply, i.e. don't require a setup.py or setup.cfg.  If that's the case, delete away.